extensions/wikia/WikiaMiniUpload/js/WMU.js

```diff
@@ -932,7 +932,7 @@ function WMU_insertPlaceholder( box ) {
 	WMU_box_filled.push(box);
 	var to_update = $( '#WikiaImagePlaceholder' + box );
 	to_update.html($( '#ImageUploadCode' ).html());
-	$.post(wgServer + wgScript + '?title=' + wgPageName  +'&action=purge');
+	$.post(wgScript + '?title=' + wgPageName  +'&action=purge');
 }
 
 function WMU_insertImage(type) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Function has a complexity of 27 which is greater than 10 authorized. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AFunctionComplexity)

[project-id]: wikia:app